tests/test_chromatic.py

```diff
@@ -275,15 +276,15 @@ def test_chromatic_add():
                                                zenith_angle=zenith_angle)
 
     # create final profile
-    final = galsim.Convolve([bdgal, chromatic_PSF])
+    final = galsim.Convolve(bdgal, chromatic_PSF)
     image = galsim.ImageD(stamp_size, stamp_size, scale=pixel_scale)
     image = final.drawImage(bandpass, image=image)
 
     do_pickle(bulge)
     do_pickle(disk)
-    do_pickle(bdgal)
+    # do_pickle(bdgal)
     do_pickle(chromatic_PSF)
-    do_pickle(final)
+    # do_pickle(final)
```

Comment:
I got this particular case back with a little bit more work in `SED.__add__`.  There are still cases that aren't picklable, of course -- anything with a python function as SED spec input for instance.